code/game/objects/items/weapons/twohanded.dm

```diff
@@ -107,6 +107,26 @@
 			qdel(A)
 			A = null
 
+/obj/item/weapon/fireaxe/attackby(obj/item/I, mob/user)
+	if(istype(I,/obj/item/tool/crowbar/halligan))
+		var/obj/item/tool/crowbar/halligan/H = I
+		to_chat(user, "<span class='notice'>You attach \the [src] and [H] to carry them easier.</span>")
+		var/obj/item/tool/irons/SI = new (user.loc)
+		SI.fireaxe = H
+		SI.halligan = src
+		user.drop_item(H)
+		H.forceMove(SI)
+		user.drop_item(src)
+		forceMove(SI)
+		user.put_in_hands(SI)
+		return 1
+	return 0
```

Comment:
no need for this trailing return if you plan to return 0
however, better to `return ..()` for safety anyway

Reply:
Resolved in latest commit